spec/support/elastic_stub.rb

```diff
@@ -0,0 +1,11 @@
+module ElasticStub
```

Comment:
Used for stubbing persistence yet still relying on ES, right now just in `describe Observation, "probably_captive?"`, however I think this could be a useful stub in the future.

Reply:
Just so I understand this, this is stubbing observations in the ES index by instantiating them and indexing them, but not committing them to the database, allowing you to test indexing without the overhead of ActiveRecord callbacks or writing the database, right? Could you explain what this is doing in a comment in the code? Because I guarantee I'm going to encounter this in a few years and forget what this does. I agree this seems like a cool pattern for these kinds of tests.

Reply:
Yeah absolutely. Classic case of it making sense when you're already deep in it, but a total mystery a few months later 😄